AzureCalling/Controllers/CallViewController.swift

```diff
@@ -205,7 +205,7 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
             cellWidth = collectionView.bounds.width / 2
             cellHeight = collectionView.bounds.height / 2
         default:
-            if UIDevice.current.orientation.isLandscape {
+            if UIScreen.main.bounds.width > UIScreen.main.bounds.height {
```

Comment:
What's the view in the ipad? does this work?

Reply:
The view still works for iPad, showing 3 x 2 (vertical x horizontal) when in portrait, and 2 x 3 when in landscape view